graylog2-web-interface/src/components/sidecars/common/SidecarSearchForm.tsx

```diff
@@ -36,35 +37,43 @@ const queryExamples = (
   </>
 );
 
-const fieldMap = {
+const fieldMap = (productName: string) => ({
   status: 'Status of the sidecar as it appears in the list, i.e. running, failing, or unknown',
   operating_system: 'Operating system the sidecar is running on',
-  last_seen: 'Date and time when the sidecar last communicated with Graylog',
+  last_seen: `Date and time when the sidecar last communicated with ${productName}`,
   node_id: 'Identifier of the sidecar',
   sidecar_version: 'Sidecar version',
-};
-
-const queryHelp = (
-  <QueryHelper entityName="sidecar" example={queryExamples} commonFields={['name']} fieldMap={fieldMap} />
-);
+});
 
 type Props = React.PropsWithChildren<{
   query: string;
   onSearch: (query: string) => void;
   onReset: () => void;
 }>;
 
-const SidecarSearchForm = ({ query, onSearch, onReset, children }: Props) => (
-  <SearchForm
-    query={query}
-    onSearch={onSearch}
-    onReset={onReset}
-    placeholder="Find sidecars"
-    queryHelpComponent={queryHelp}
-    topMargin={0}
-    useLoadingState>
-    {children}
-  </SearchForm>
-);
+const SidecarSearchForm = ({ query, onSearch, onReset, children = undefined }: Props) => {
+  const productName = useProductName();
+  const queryHelp = (
+    <QueryHelper
+      entityName="sidecar"
+      example={queryExamples}
+      commonFields={['name']}
+      fieldMap={fieldMap(productName)}
+    />
+  );
```

Comment:
We can move this part outside of the component into a separate component and then use `<SidecarQueryHelper />` as a props `queryHelpComponent`  in SearchForm.

Reply:
Sounds good.